### DISCUSSION TITLE
Generate data class from table

### DISCUSSION THREAD

**Discussion by kadirgun** [spatie/laravel-data#767] [Q&A]
Is there any way to create a data file from a table or migration file?

For example:
```bash
php artisan make:data --table users
```
This command will pull columns from the users table and create a UserData.php file.

**Comment by anburocky3** [comment]
It would be great feature. Doing it manually is tedious.

**Comment by carlin-rj** [comment]
I have implemented this function, but it is only for personal use and I have not gone into a general design. I made some code adjustments with reference to barryvdh/laravel-ide-helper. Maybe you can also be inspired by this
